### ISSUE TITLE
Data of Type F5Telemetry was dropped because number of field is above the limit of 500

### ISSUE THREAD

**Issue by ilhamsetiapambudi** [F5Networks/f5-telemetry-streaming#156]
### Environment
 * Telemetry Streaming Version:
 * BIG-IP Version: 15.1.2.1
 * TS Version: 1.20.1
 * AS3 Version: 3.29.0

### Summary

1. We will be telemetry streaming LTM and ASM to Azure Log Analytic.
2. When performing Telemetry Streaming LTM and ASM all logs can be received in Azure Log Analytics.
3. When **adding** configuration for **Telemetry Streaming AVR**, Azure Log analytics has an error **Data of Type F5Telemetry was dropped because number of field is above the limit of 500** on _F5Telemetry_clientSslProfiles, F5Telemetry_serverSslProfiles, F5Telemetry_pools and F5Telemetry_virtualServers._
4. The last condition is reverted to the original telemetry streaming config for **LTM and ASM**
5. Error **Data of Type F5Telemetry was dropped because number of field is above the limit of 500** still appears

### Steps To Reproduce
Declaration JSON using POSTMAN.

1. Submit the following declaration:
```json
{
  "class": "Telemetry",
  "controls": {
    "class": "Controls",
    "logLevel": "info"
  },
  "My_Poller": {
        "class": "Telemetry_System_Poller",
        "interval": 60
    },
  "My_Listener": {
    "class": "Telemetry_Listener",
    "port": 6514
  },
  "My_Consumer": {
    "class": "Telemetry_Consumer",
    "type": "Azure_Log_Analytics",
    "workspaceId": "***",
    "passphrase": {
      "cipherText": "***"
      }
  }
}
```

### Expected Behavior
All LTM, ASM and AVR logs will be logged in Azure Log Analytics and not exceed 500 fields limit.

### Actual Behavior
Error **Data of Type F5Telemetry was dropped because number of field is above the limit of 500** on _F5Telemetry_clientSslProfiles, F5Telemetry_serverSslProfiles, F5Telemetry_pools and F5Telemetry_virtualServers._

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-2701.

**Comment by ilhamsetiapambudi** [comment]
For solve the problem, what should I do? Thank you

**Comment by dstokesf5** [comment]
I believe the issue is due to how Azure is processing the JSON payloads. I believe we will  need to add an option to send data in an alternative format to Azure to resolve this issue.

In the meantime, you can try reducing the amount of data sent to Azure using Telemetry Streaming's Data Modification features: https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/data-modification.html

**Comment by ilhamsetiapambudi** [comment]
Now I have done a telemetry data filter to Azure, but there is still data that is dropped by Azure. And here is the data filter that I use :

"actions": [
            {
                "enable": true,
                "includeData": {},
                "locations": {
                    "virtualServers": true
                }
            }

**Comment by dstokesf5** [comment]
This has been resolved in the 1.24.0 release.
